src/GrpcProtos/capture.proto

```diff
@@ -738,6 +750,8 @@ message ClientCaptureEvent {
     FunctionCall function_call = 2;
     GpuJob gpu_job = 3;
     GpuQueueSubmission gpu_queue_submission = 4;
+    WarningInstrumentingWithUserSpaceInstrumentationEvent
+        warning_instrumenting_with_user_space_instrumentation_event = 48;
```

Comment:
Move down to keep the alphabetical order.

Reply:
done